src/main/kotlin/bandcampcollectiondownloader/core/BandcampCollectionDownloader.kt

```diff
@@ -143,6 +146,18 @@ class BandcampCollectionDownloader(private val args: Args, private val io: IO) {
             return
         }
 
+        // Filter by Artist
+        if (args.filterArtist != null && digitalItem.artist != args.filterArtist) {
+            cache.add(saleItemId, "UNKNOWN")
+            return
+        }
+
+        // Filter by Title
+        if (args.filterTitle != null && digitalItem.title != args.filterTitle) {
+            cache.add(saleItemId, "UNKNOWN")
+            return
+        }
+
```

Comment:
Not sure why you are adding things to the cache file here? Doesn't it mean that the releases become "blacklisted" somehow? 

In my view, the goal of this cache file was to fully reflect the list of releases that are currently already present in the folder. To keep it that way, I would tend to prefer that we avoid using `cache.add` here.

Maybe another way to deal with this would be to be able to run the tool with a configuration file, in which one could enter all its filtering criteria, and reuse this file as much as needed when re-rerunning the tool in a given folder? I'll think about that :)